packages/output/src/utils/backgroundAudio.js

```diff
@@ -43,6 +43,7 @@ function BackgroundAudio({ backgroundAudio, id }) {
         autoPlay="autoplay"
         layout="nodisplay"
         poster=""
+        crossorigin="anonymous"
```

Comment:
We could optionally put this attribute on if needed?

Reply:
I thought about it, but there's no harm in always adding it. Avoids having to add extra complexity for that.